src/vlogger/vlogger.h

```diff
@@ -359,20 +361,13 @@ void vlog_stop(void);
 
 static inline uint32_t vlog_get_usec_since_start()
 {
-    struct timespec ts_now;
-
-    BULLSEYE_EXCLUDE_BLOCK_START
-    if (gettime(&ts_now)) {
-        printf("%s() gettime() Returned with Error (errno=%d %m)\n", __func__, errno);
-        return (uint32_t)-1;
-    }
-    BULLSEYE_EXCLUDE_BLOCK_END
+    auto now = steady_clock::now();
 
-    if (!g_vlogger_usec_on_startup) {
-        g_vlogger_usec_on_startup = ts_to_usec(&ts_now);
+    if (g_vlogger_usec_on_startup.time_since_epoch() == steady_clock::duration::zero()) {
+        g_vlogger_usec_on_startup = now;
     }
 
-    return (ts_to_usec(&ts_now) - g_vlogger_usec_on_startup);
+    return duration_cast<microseconds>(now - g_vlogger_usec_on_startup).count();
```

Comment:
does uint32_t is enough to hold us.count()?
what heppens if .count() is > max uint32_t?

its a general question for all the code.

Reply:
> does uint32_t is enough to hold us.count()?

No

> what heppens if .count() is > max uint32_t?

At worst the comiler will look at the 32 least significant bits, same as now.